src/molecules/header.tsx

```diff
@@ -33,16 +33,12 @@ const HeaderWrapper = styled.div`
   padding-top: 2%;
   @media (max-width: ${({ theme }) => theme.breakpoints.sm + 'px'}) {
     width: 80%;
-    padding-top: 0%;
-    padding-bottom: 10%;
+    margin-top: 5vh;
```

Comment:
Does this offset depend on the view height? Maybe it's better to offset this by the height of the filter bar?

Reply:
Good thinking, I put It to 50px instead